app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  has_many :likes, foreign_key: :post_id
+  has_many :comments, foreign_key: :post_id
```

Comment:
- [ ] [REQUIRED] I can also see that there's an addition of foreign keys in your model associations (has_many :likes, foreign_key: :post_id and has_many :comments, foreign_key: :post_id). However, based on the provided ERD, these foreign keys are not part of the intended structure.

Please remove the foreign keys and update your model associations to adhere to the correct structure.